pkg/authorization/bearer_token_test.go

```diff
@@ -0,0 +1,57 @@
+package authorization_test
+
+import (
+	"os"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+
+	"github.com/kubevirt/hyperconverged-cluster-operator/pkg/authorization"
+)
+
+var _ = Describe("BearerToken", func() {
+	BeforeEach(func() {
+		tmpFile, err := os.CreateTemp("", "fake-token")
+		Expect(err).ToNot(HaveOccurred())
+
+		_, err = tmpFile.Write([]byte("test-secret-key"))
+		Expect(err).ToNot(HaveOccurred())
+
+		os.Setenv("KUBERNETES_SERVICE_TOKEN_PATH", tmpFile.Name())
```

Comment:
Please use the `tokenPathEnvVar` const, or if you don't want to expose it, redefine it here.

Reply:
done